src/containers/NFT/NFT.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import NoMatch from '../NoMatch'
+import NFTHeader from './NFTHeader/NFTHeader'
+import './styles.scss'
+import {
+  analytics,
+  ANALYTIC_TYPES,
+  NOT_FOUND,
+  BAD_REQUEST,
+} from '../shared/utils'
+
+const ERROR_MESSAGES: any = {
+  [NOT_FOUND]: {
+    title: 'assets.no_nfts_message',
+    hints: ['check_nft_id'],
+  },
+  [BAD_REQUEST]: {
+    title: 'invalid_xrpl_address',
+    hints: ['check_nft_id'],
+  },
+}
+
+const DEFAULT_ERROR: any = {
```

Comment:
is typescript complaining if you don't type this as `any`? use of `any` is not good. if you need to type this:

```typescript
interface ErrorMessage {
  title: string,
  hints: string[],
}

const ERROR_MESSAGES: { [string]: ErrorMessage } = ...
```